openslides_gui/gui.py

```diff
@@ -0,0 +1,763 @@
+import datetime
+import errno
+import gettext
+import itertools
+import json
+import locale
+import os
+import queue
+import subprocess
+import sys
+import threading
+
+import psutil
+
+import wx
+import wx.adv
+
+import openslides
+
+from openslides.utils.main import (
+    detect_openslides_type,
+    get_default_user_data_path,
+    PortableDirNotWritable,
+)
+
+
+# NOTE: djangos translation module can't be used here since it requires
+#       a defined settings module
+_translations = gettext.NullTranslations()
+_ = lambda text: _translations.gettext(text)
+ngettext = lambda msg1, msg2, n: _translations.ngettext(msg1, msg2, n)
```

Comment:
That won't work since the _translations object will be replaced after the translation are loaded (not that we currently have translations)
I did however remove the obsolete comment and change the location from which translations would be loaded (during early devolpment it was an submodule of openslides and would have reused it's translations, but that never went live)